--- configure.ac.keep	2015-12-28 22:08:31.054969617 +0200
+++ configure.ac	2015-12-28 22:15:09.613521558 +0200
@@ -33,7 +33,7 @@
     [Set CUDA install dir @<:@]m4_default(/usr/local/cuda/)[@:>@]
   ),
   [cuda_home=$withval], 
-  [cuda_home=/usr/local/cuda/]
+  [cuda_home=/usr]
 )
 
 
@@ -105,9 +105,9 @@
   NVCC="${cuda_home}/bin/nvcc -gencode arch=compute_13,code=sm_13"
   
   RTH_CPPFLAGS="-DTHRUST_DEVICE_SYSTEM=THRUST_DEVICE_BACKEND_CUDA"
-  RTH_LDFLAGS="-L${cuda_home}/lib/ -lcuda -lcudart"
+  RTH_LDFLAGS="-L${cuda_home}/lib64/ -lcuda -lcudart"
   
-  CUDA_INCS="-I\"${cuda_home}/include\" -I\"${R_INCLUDE_DIR}\" -I\"../inst/include/\""
+  CUDA_INCS="-I\"${cuda_home}/include/cuda\"  -I\"${R_INCLUDE_DIR}\" -I\"../inst/include/\""
 else
   AC_MSG_ERROR([Unknown backend; use one of OPENMP, TBB, or CUDA.])
 fi
